apps/landing/styles.css

```
@@ -0,0 +1,717 @@
+.br-test-list { list-style: none; margin: 0; padding: 0; display: flex; flex-direction: column; gap: 4px; }
+.br-test-row { display: grid; grid-template-columns: 18px 1fr; gap: 10px; align-items: baseline; padding: 8px 10px; border: 1px solid var(--line); border-radius: var(--radius); background: var(--bg-card); transition: background 200ms ease-out; }
+.br-test-row[data-state="pass"] { border-color: color-mix(in srgb, #2f9e6d 55%, var(--line)); background: color-mix(in srgb, #2f9e6d 7%, var(--bg-card)); }
+.br-test-row[data-state="warn"] { border-color: color-mix(in srgb, #c9a227 55%, var(--line)); background: color-mix(in srgb, #c9a227 7%, var(--bg-card)); }
+.br-test-row[data-state="fail"] { border-color: color-mix(in srgb, #c14b4b 55%, var(--line)); background: color-mix(in srgb, #c14b4b 7%, var(--bg-card)); }
+.br-test-mark { font-family: var(--mono); font-size: 14px; text-align: center; line-height: 1; }
+.br-test-row[data-state="pass"] .br-test-mark { color: #2f9e6d; }
+.br-test-row[data-state="warn"] .br-test-mark { color: #c9a227; }
+.br-test-row[data-state="fail"] .br-test-mark { color: #c14b4b; }
+.br-test-body { display: flex; flex-direction: column; gap: 2px; min-width: 0; }
+.br-test-label { font-size: 12.5px; color: var(--ink); }
+.br-test-note { font-size: 11.5px; color: var(--ink-3); line-height: 1.4; word-break: break-word; }
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)